style.css

```diff
@@ -0,0 +1,118 @@
+body {
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 150px;
+    background-color: papayawhip;
+    accent-color: greenyellow;
+
+}
+
+form {
+    border-radius: 30px;
+    background-color: #687fbe;
+}
+
+fieldset {
+    border-radius: 4px;
+    border: 3px solid papayawhip;
+    margin: 25px 30px 25px 30px;
+    padding: 20px 20px 20px 20px;
+}
+
+legend {
+    padding-left: 15px;
+    padding-right: 15px;
+    font-size: 30px;
+    font-weight: 700;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
=D 
First time I see such a huge amount of fallbacks. Well your text will look nice even on Nokia 3310